spec/models/authority_spec.rb

```diff
@@ -1,6 +1,16 @@
 require 'spec_helper'
 
 describe Authority do
+  describe "validations" do
+    it 'should ensure a unique short_name' do
+      @a1 = create(:authority, short_name: "Existing")
+      @a2 = build(:authority, short_name: "Existing")
+
+      @a2.valid?.should == false
+      @a2.errors.messages[:short_name].should == ["has already been taken"]
```

Comment:
Let's change these to the newer `expect` syntax here. I realise that a few of the other tests in this file use the `should` syntax, but we're using the newer syntax for new tests and trying to update old ones when we touch them. http://betterspecs.org/#expect